Shoko.Server/Mappings/AniDB_TagMap.cs

```diff
@@ -11,12 +11,13 @@ public AniDB_TagMap()
         Not.LazyLoad();
         Id(x => x.AniDB_TagID);
 
-        Map(x => x.GlobalSpoiler).Not.Nullable();
-        Map(x => x.LocalSpoiler).Not.Nullable();
-        Map(x => x.Spoiler).Not.Nullable();
-        Map(x => x.TagCount).Not.Nullable();
-        Map(x => x.TagDescription).Not.Nullable().CustomType("StringClob");
         Map(x => x.TagID).Not.Nullable();
-        Map(x => x.TagName).Not.Nullable();
+        Map(x => x.ParentTagID);
+        Map(x => x.TagNameSource).Column("TagName").Not.Nullable();
```

Comment:
I would name it TagName and TagNameOverride. The Override part is a modifier.

Reply:
I didn't want to update all the existing occurrences of `TagName` in the code to a new getter so instead did i create a new property for the source name and mapped it to the database field, and then made the original property a getter (only) that gets the override or the source if no override is available. This way we don't need to update 34 (iirc) occurrences of `TagName`.

Reply:
I see. That is acceptable